### PR TITLE
KAFKA-2774: Rename Copycat to Kafka Connect

### PR DESCRIPTION
**Pull request by ewencp**

### CODE REVIEW

File: bin/connect-distributed.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
+# this work for additional information regarding kright ownership.
```
Comment (gwenshap) [inline bin/connect-distributed.sh]: LOL.
Careful with your sed command :)

Comment (ewencp) [inline bin/connect-distributed.sh]: Haha, not even sed, just some random editor typo. You managed to catch that just before I pushed the fix :)

**Comment by gwenshap** [comment]
LGTM

File: config/connect-standalone.properties [hunk 2]
```diff
@@ -27,11 +27,11 @@ value.converter.schemas.enable=true
 
 # The internal converter used for offsets and config data is configurable and must be specified, but most users will
 # always want to use the built-in default. Offset and config data is never visible outside of Copcyat in this format.
```
Comment (wushujames) [inline config/connect-standalone.properties:29]: Copcyat->Copycat->Connect

File: bin/connect-distributed.sh (hunk 1 above)
Comment (wushujames) [inline bin/connect-distributed.sh]: Lol.

**Pull request closed without merging**